src/backend/egl/display.rs

```diff
@@ -1110,3 +1136,24 @@ pub struct PixelFormat {
     /// is srgb enabled
     pub srgb: bool,
 }
+
+/// Denotes if damage tracking is supported.
+///
+/// Additionally notes which variant of the `EGL_*_swap_buffers_with_damage` extension was found.
+/// Prefers `KHR` over `EXT`, if both are available.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum DamageSupport {
```

Comment:
One final thing, does this need to be public?

Reply:
Yes, it is part of the NativeSurface trait signature.